package/root/usr/local/sbin/resize_rootfs.sh

```diff
@@ -2,6 +2,13 @@
 
 set -xe
 
+if [ "$(id -u)" -ne "0" ]; then
+	echo "This script requires root."
+	exit 1
+fi
+
+apt-get update && apt-get install gdisk parted
```

Comment:
Yes, please add them to these lists, but also add them here: https://github.com/ayufan-rock64/linux-build/blob/master/rootfs/make_rootfs.sh#L159.

And add them here: https://github.com/ayufan-rock64/linux-build/blob/master/Makefile#L38 as `--depends <package>`.